src/api/app/models/branch_package.rb

```diff
@@ -224,6 +224,10 @@ def create_branch_project
         tprj.relationships.build(user: User.current, role: Role.find_by_title!('maintainer'))
         tprj.flags.create(flag: 'build', status: 'disable') if @extend_names
         tprj.flags.create(flag: 'access', status: 'disable') if @noaccess
+        if params[:project]
+          project = Project.get_by_name(params[:project])
+          tprj.flags.create(flag: 'publish', status: 'disable') if project.try(:image_template?)
```

Comment:
Depending on the Configuration this is already done here:
https://github.com/openSUSE/open-build-service/blob/c1b60cf8675813a901a93e35d2720c7462a729fa/src/api/app/models/project.rb#L1278

Maybe it's is worth to move it there...